Dockerfile

```
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
```

```suggestion
FROM mcr.microsoft.com/dotnet/sdk:6.0
```

Removed the `AS` keyword since this is used for multi-stage builds and is superfluous in this context.